README.md

```diff
@@ -344,3 +344,27 @@ Use the `subscriptionRoleBinding` subcommand to remove a role binding from an ac
 * For the given account, delete a role binding created at application level using the roleID `ocm support delete applicationRoleBinding [accountID] [roleID]`
 * For the given account, delete a role binding created at organization level using the roleID `ocm support delete organizationRoleBinding [accountID] [orgID] [roleID]`
 * For the given account, delete a role binding created at subscription level using the roleID `ocm support delete subscriptionRoleBinding [accountID] [subscriptionID] [roleID]`
+
+
+### Patch
+
+The `patch` command patches the given resource.
+
+#### Patching subscriptions
+
+Use the `subscriptions` subcommand to patch a subscription by passing the ID, or provide filter value to search matching subscriptions and patch them. Pass the JSON body for the patch request in terminal using `echo '{<PATCH_BODY>}' | ` before the actual command. By default the dry run flag will be enabled. Pass `dry-run=false` flag to actually remove the resource.
+
+The following flags are available for `delete capability`:
+
+```
+--filter                     If non-empty, filters and patches the matching subscriptions.
+--dryRun                     If false, it will execute the patch command call in instead of a dry run.
+--maxRecords                 Ensures that the maximum number of resources on which the operation is performed does not exceed than maxRecords, when passed dryRun as false. Defaults to 100.
```

Comment:
Can you review/simplify this description?

Reply:
Changed to "When dryRun is false, ensures that the maximum number of affecting records do not exceed maxRecords"

Reply:
Did you mean "affected" records?

Here is what I had in mind: Maximum number of affected records. Defaults to 100. Only effective when dryRun is set to false.

Reply:
Changed